semgrep-core/src/fixing/Autofix_printer.mli

```diff
@@ -0,0 +1,10 @@
+(* Print the given AST to a string, using the original text of nodes from the
+ * fix pattern or target file wherever possible *)
+val print_ast :
+  lang:Lang.t ->
+  metavars:Metavariable.bindings ->
+  target_contents:string Lazy.t ->
+  fix_pattern_ast:AST_generic.any ->
+  fix_pattern:string ->
```

Comment:
why do we pass also the fix_pattern?
What's the difference between the AST_generic.any passed below and the fix_pattern_ast argument?

Reply:
See my comment above